specs/CoreWebView2.CreateFromCOMObject.md

```diff
@@ -17,8 +17,14 @@ To help implement this for Unity, we are adding a new static factory function on
 class that will allow it to wrap an existing ICoreWebView2 COM object, instead of creating a new
 one.
 
+To round out the more general scenario of interoperating between libraries written in different
+languages, we add the ability to convert a CoreWebView2 to and from .NET and COM, as well as
+to and from WinRT and COM.
+
```

Comment:
I'm missing a premise: are there features of COM ICoreWebView2 that aren't exposed by WinRT [CoreWebView2](https://learn.microsoft.com/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/CoreWebView2)?

Reply:
There are some yes but also its so that you can mix code (esp libraries) written in COM / WinRT / .NET